Source/Pawnmorphs/Esoteria/Hediffs/IMutationHediff.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Pawnmorph.Hediffs
+{
+    /// <summary>
+    /// Interface for all hediffs that can possibly cause mutations
+    /// </summary>
+    public interface IMutationHediff
```

Comment:
IMutagenicHediff would be a better name. small thing but would help avoid confustion with Hediff_AddedMutation

Reply:
Whoops, I thought that was the name I gave it lol.  Will change it